modules/apps/static/portal-configuration/portal-configuration-persistence/src/main/java/com/liferay/portal/configuration/persistence/ConfigurationPersistenceManager.java

```diff
@@ -51,10 +54,16 @@
 import org.apache.felix.cm.PersistenceManager;
 import org.apache.felix.cm.file.ConfigurationHandler;
 
+import org.osgi.framework.BundleContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Modified;
+
 /**
  * @author Raymond Augé
  * @author Sampsa Sohlman
  */
+@Component(immediate = true)
```

Comment:
Never mind, I figured out it was to initialize the serviceTracker for the listeners.